addons/storyshots/storyshots-core/README.md

```diff
@@ -185,6 +185,21 @@ initStoryshots({
 })
 ```
 
+Provide a function to have story-specific options:
```

Comment:
It looks like this is related to a previous section (Using `createNodeMock` to mock refs), also please mention that it works with `multiSnapshotWithOptions` (it should, right ?)